src/languages.py

```diff
@@ -0,0 +1,37 @@
+"""
+Каждый из N школьников некоторой школы знает Mi языков. Определите, какие языки
+знают все школьники и языки, которые знает хотя бы один из школьников.
+"""
+school_child_count = int(input("Please enter the number of school_child:"))
+# создание словаря, в котором ключ - условный порядковый номер ученика,
```

Comment:
in fact, redundant comments.  good naming and code would be enough documentation